public/css/main.css

```diff
@@ -4,21 +4,27 @@ html {
   font-size: 62.5%; /* 1rem = 10px */
 }
 
-body {
+body, div, span, a {
   margin: 0;
   padding: 0;
   border: 0;
   box-sizing: border-box;
 }
 
-a:hover, button:hover {
+a:hover, button:hover, .cal-nav-button, td {
   cursor: pointer;
 }
 
 a {
   text-decoration: none;
 }
 
+div, tr, td, span {
+  display: flex;
+  justify-content: center;
```

Comment:
are you sure you want everything to have display flex?
and what for ?

Reply:
Makes positioning everything a lot easier